.gitignore

```diff
@@ -20,6 +20,8 @@ Makefile.in
 
 .diags.log.meta
 
+.vscode/
```

Comment:
Yes, this should be cleaned up. It looks like an accident since I don't see how it got in to `Makefile.in` without being in `Makefile.am`.